javascript/src/createMeta.ts

```diff
@@ -1,10 +1,18 @@
-import os from 'os'
-import { parse as parseUrl, format as formatUrl } from 'url'
 import * as messages from '@cucumber/messages'
+import os from 'os'
+import { format as formatUrl, parse as parseUrl } from 'url'
+
 import defaultCiDict from './ciDict.json'
 import evaluateVariableExpression from './evaluateVariableExpression'
 import { CiDict, CiSystem, Env } from './types'
 
+interface gitHash {
```

Comment:
Use a capital letter for types. Prefer `type` over `interface` when there are no methods. `gitHash` sounds strange to me. How about `GitInfo`?

Reply:
(Aside: Perhaps a [Git "tree-ish"](https://git-scm.com/docs/gitglossary#def_tree-ish)? But, the remote is not a part of that, though.)

Reply:
Indeed it looks a little bit like a tree-ish

GitInfo seems more generic, and maybe more common in a context like the one in CreateMeta.

(I was not convince with gitHash myself :p)